### ISSUE TITLE
Upload Sourcemap build script phase fails if using yarn workspaces

### ISSUE THREAD

**Issue by jtoce** [Instabug/Instabug-React-Native#643]
### Steps to Reproduce the Problem

1. Create root directory `my-project`
2. Configure [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces)
```
{
  "name": "my-project",
  "version": "1.0.0",
  "private": true,
  "workspaces": {
    "packages": [
      "my-app",
      "my-library",
    ]
  }
}
```
3. Create new React Native app `npx react-native init MyApp --directory my-app`
4. Create new folder `my-library` and `yarn init`
5. `cd my-app`
6. Install Instabug `yarn add instabug-reactnative`
7. Install Cocoapods `cd ios` and `pod install`
8. `cd ..`
9. Run ios `yarn react-native run-ios`

Final directory structure looks like this:
```
--my-project
  |-my-app
  |-my-library
  |-node_modules
    |-instabug-reactnative
```
NOTE: In yarn workspaces- `node_modules` is a level deeper than usual, since it's at the workspace root.

### Actual Behavior

File path of `node_modules` is hard coded, causing build phase to fail.

```
Running script '[CP-User] [instabug-reactnative] Upload Sourcemap'
bash: ../node_modules/instabug-reactnative/ios/upload_sourcemap.sh: No such file or directory
Command PhaseScriptExecution failed with a nonzero exit code
```

### Expected Behavior

instabug-reactnative detects the location of the `node_modules` folder and correctly links the `upload_sourcemap.sh` script.

### React Native SDK Version
10.9.1

**Comment by jtoce** [comment]
Also- in the file: https://github.com/Instabug/Instabug-React-Native/blob/master/link.rb
```
upload_build_phase.run_only_for_deployment_postprocessing = '1'
```
But after running `pod install` in the project file:
```
runOnlyForDeploymentPostprocessing = 0;
```
Not sure if that's being correctly set.

**Comment by AliAbdelfattah** [comment]
@jtoce thank you for the detailed reproduction steps, we're looking into it.

**Comment by DavidMina96** [comment]
@jtoce The fix is released in v11.0.0.
